clang/lib/Driver/Driver.cpp

```diff
@@ -7758,9 +7758,17 @@ Action *Driver::BuildOffloadingActions(Compilation &C,
     return HostAction;
   }
 
+  // For SYCL offloading with -fsycl-host-compiler enabled, we do not have the
+  // ability to embed the packaged file.
```

Comment:
Can you share info on why we do not have the ability to embed the packaged file?

Reply:
The clang based compiler has the ability to embed external binaries via the `-fembed-offload-object` option.  This occurs when generating the host object.  When we are using a 3rd party compiler to create the host object, there is no way to embed the device object into it during compile time, so we are using bundles instead (i.e. the old model fat object)